operator/internal/controller/auto/workspace_controller.go

```diff
@@ -433,10 +433,8 @@ func fqdnForService(w *autov1alpha1.Workspace) string {
 
 func labelsForStatefulSet(w *autov1alpha1.Workspace) map[string]string {
 	return map[string]string{
-		"app.kubernetes.io/name":       "pulumi",
-		"app.kubernetes.io/component":  "workspace",
-		"app.kubernetes.io/instance":   w.Name,
-		"app.kubernetes.io/managed-by": "pulumi-kubernetes-operator",
```

Comment:
Is there a rationale for removing the well known `app.kubernetes.io/managed-by` label? This makes it obvious to other tools what is managing these child objects.

Reply:
I was hesitant to use a subset of app labels, seemed a bit weird.  The definition of the `app.kubernetes.io/managed-by` label is:
> The tool being used to manage the operation of an application.

To me, it identifies the tool that is servicing the application at large.  Since PKO doesn't know the application to which the workload is associated, it should not apply this label.

Reply:
> Since PKO doesn't know the application to which the workload is associated, it should not apply this label.

I disagree. The objects we are creating here (Workspace, Statefulset) are all created by our own PKO and should also be managed by PKO. These objects are what enables PKO to work right now. What happens if another controller/tool takes over control of these objects? We'd likely get into a controller fight here. Though practically, these labels are most generally used for metrics, so having the `app.kubernetes.io/managed-by` would allow users to find metrics for PKO created/managed objects in their current dashboards.

Reply:
They could also use the pulumi.com labels to identity these pods.

If you were to deploy an EKS cluster with various addons, I would guess that you would not see `app.kubernetes.io/managed-by` on the majority of the objects. It just isn't as common as you're suggesting.